### PR TITLE
fix: wds modal resizer

### PR DESCRIPTION
**Pull request by KelvinOm**
## Description
- Move canvas resizer by 1 DOM node higher.
- Used `contain: strict;` for modal styles.

**Before:**
Снимок экрана 2024-02-20 в 17 45 30

**After:**
Снимок экрана 2024-02-21 в 17 58 05

## Summary by CodeRabbit

- **Style**
	- Improved modal overlay containment with updated CSS properties.
	- Enhanced page alignment with new centering style for the `PageViewWrapper` component.

- **Refactor**
	- Streamlined canvas viewport handling in the Widgets Editor for better conditional styling.
- **Chores**
	- Updated Cypress configuration to modify scroll behavior settings.

### CODE REVIEW

File: app/client/src/pages/Editor/WidgetsEditor/MainContainerWrapper.tsx [hunk 1]
```diff
@@ -184,7 +183,7 @@ function MainContainerWrapper(props: MainCanvasWrapperProps) {
           isProtectedMode ||
           isAppSettingsPaneWithNavigationTabOpen
             ? isWDSV2Enabled
-              ? "var(--bg-color)"
```
Comment (jsartisan) [inline app/client/src/pages/Editor/WidgetsEditor/MainContainerWrapper.tsx:187]: are we sure about this change? don't we want bg from color algorithm on wds/anvil apps?

File: app/client/src/pages/Editor/WidgetsEditor/MainContainerWrapper.tsx [hunk 2]
```diff
@@ -203,7 +202,6 @@ function MainContainerWrapper(props: MainCanvasWrapperProps) {
             !isAnvilLayout,
           "mt-24": shouldShowSnapShotBanner,
         })}
-        id={CANVAS_VIEWPORT}
```
Comment (jsartisan) [inline app/client/src/pages/Editor/WidgetsEditor/MainContainerWrapper.tsx:206]: I hope this won't alter fixed layout.

File: app/client/src/pages/Editor/WidgetsEditor/MainContainerWrapper.tsx (hunk 1 above)
Comment (KelvinOm) [reply app/client/src/pages/Editor/WidgetsEditor/MainContainerWrapper.tsx:187]: MainContainerWrapper is located above provider, there is no access to this variable. Provider also sets the background out of the box.

File: app/client/src/pages/Editor/WidgetsEditor/MainContainerWrapper.tsx (hunk 2 above)
Comment (KelvinOm) [reply app/client/src/pages/Editor/WidgetsEditor/MainContainerWrapper.tsx:206]: Everything seems to be working there :crossed_fingers:

File: app/client/cypress_ci.config.ts [hunk 3]
```diff
@@ -17,7 +17,7 @@ export default defineConfig({
   chromeWebSecurity: false,
   viewportHeight: 1200,
   viewportWidth: 1400,
-  scrollBehavior: "center",
+  scrollBehavior: false,
```
Comment (albinAppsmith) [inline app/client/cypress_ci.config.ts:20]: Can you remove these changes? this will cause more failures for the tests which assume it is scrolled already.

**Pull request merged**